### PR TITLE
puts bullets in the security techfab (they aren't free)

### PR DESCRIPTION
**Pull request by dinnerwoerror**
## About The Pull Request

i think it's stupid that you cant get dinky little handgun cartridges without spending big bucks on a super illegal emag type thing, so why should a man not just shoot up the security department and use their techfab to make more ammunition for themselves?
## Why It's Good For The Game

you can now get your hands on a security techfab to make more ammunition (note, there are only 2 that are in existence on the station, and it's really hard to find another one off the station)

## Changelog
:cl:
add: the security techfab has the functionality of making a handful of very common handgun calibers, although it's still wedged inside ballistic technology, so fat chance of you ever actually building any
/:cl:

### CODE REVIEW

File: code/modules/research/designs/weapon_designs.dm [hunk 1]
```diff
@@ -296,13 +296,40 @@
...
+	name = ".357 Casing"
+	id = "a357"
 	build_type = PROTOLATHE
 	materials = list(/datum/material/iron = 4000)
-	build_path = /obj/item/ammo_box/magazine/co9mm
+	build_path = /obj/item/ammo_casing/a357
+	category = list("Ammo")
+	departmental_flags = DEPARTMENTAL_FLAG_SECURITY
+
+/datum/design/c10mm //WS edit - sideline lethals
```
Comment (Maldaris) [inline code/modules/research/designs/weapon_designs.dm:308]: I like most of this, but my one gripe is we're including Stetchkin-caliber ammo in the lathe. Unless I'm mistaken most of the ammo in the tator arsenal is 10mm? Are there any examples of guns that aren't?

Comment (dinnerwoerror) [reply code/modules/research/designs/weapon_designs.dm:308]: the surplus rifle
the projectile pistol
that's literally about it

Comment (Maldaris) [reply code/modules/research/designs/weapon_designs.dm:308]: Surplus rifle is Jake from State Farm's primary, and the pistol is again, a syndicate item. I'd recommend we take this out of the normal techweb and make it part of the emag set (if it's not already).

Comment (dinnerwoerror) [reply code/modules/research/designs/weapon_designs.dm]: what 10mm ammo? it never existed

**Review by Superyodeler — CHANGES_REQUESTED** [review]
I'm assuming the 9mm box is meant for the stetchkin, but the stetchkin uses 10mm ammo.

**Comment by dinnerwoerror** [comment]
ok and what about it

**Comment by dinnerwoerror** [comment]
if you want the 10 millimeter ammunition say yes or no

**Comment by Baldir-Odensen** [comment]
The 9mm ammo could be for the APS.

Can't remember if there is a method of getting aps bullets other than lootbox crate possibly

**Comment by TheNeoGamer42** [comment]
9mm is for commanders too I think, which it also looks like you didn't keep the commander mag recipe

**Pull request closed without merging**